index.html

```diff
@@ -0,0 +1,124 @@
+<!--Mobile version of template 1-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Mobile Version</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--Work section-->
+    <section class="work">
+        <!---The first card -->
+        <div class="flex">
+            <img src="images/card1-image.png" alt="Image for Card 1">
+            <h2>TONIC</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+         <!---The second card -->
+         <div class="flex">
+            <img src="images/card2-image.png" alt="Image for Card 2">
+            <h2>TONIC</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
```

Comment:
- [ ] Nice working on your `cards`, kindly use a `p` tag for The paragraph ("A daily selection of privately personalized reads; no accounts or sign-ups required.") for better use of HTML semantics.
- [ ] Also kindly change the `color` and reduce the `font-size` of the list of project technologies to match what you have on the Figma Template.

### Figma Design
![is-figma](https://user-images.githubusercontent.com/48371156/176654647-26d96305-9f85-4d7a-acfb-f4041a7d3361.png)

### Your Design
![is-worked](https://user-images.githubusercontent.com/48371156/176654658-bce757aa-950a-48e9-a545-b86614c4010b.png)